src/api/routes/likeDislikeCommentRoute.js

```diff
@@ -0,0 +1,22 @@
+import { Router } from 'express';
+import likeDislikeController from '../controllers/likeDislikeController';
+import checkValidToken from '../../middlewares/checkValidToken';
+import errorHandler from '../../middlewares/errorHandler';
+import likeDislikeCheck from '../../middlewares/likeDislikeCheck';
+
+const likeDislikeRouter = new Router();
+likeDislikeRouter.post('/like/:commentId',
+  checkValidToken,
+  likeDislikeCheck.liked,
+  errorHandler(likeDislikeController.likeComment));
+likeDislikeRouter.post('/dislike/:commentId',
+  checkValidToken,
+  likeDislikeCheck.disliked,
+  errorHandler(likeDislikeController.dislikeComment));
+likeDislikeRouter.get('/dislike/:commentId',
+  checkValidToken,
+  errorHandler(likeDislikeController.getDislikeCount));
```

Comment:
what does this function perform here @murediane?

Reply:
what functions